superset-frontend/src/explore/components/controls/DateFilterControl/DateFilterControl.tsx

```diff
@@ -76,12 +76,12 @@ const fetchTimeRange = async (
   const endpoint = `/api/v1/time_range/?q=${query}`;
   try {
     const response = await SupersetClient.get({ endpoint });
-    const timeRangeString = buildTimeRangeString(
-      response?.json?.result?.since || '',
-      response?.json?.result?.until || '',
-    );
+    const { since = '', until = '' } = response?.json?.result || {};
+    const timeRangeString = buildTimeRangeString(since, until);
```

Comment:
NEED to double check here.
`timeRangeString` is `Absolute time range`. 
eg:
if today is 2021-2-8, then `last day`, generate "2021-02-07 : 2021-02-08".
when the 2021-3-8, then `last day` generate "2021-03-07 : 2021-03-08"